src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -48,6 +53,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Post newPost = new Post(locationName, category, parking, ratingScore, noiseScore, spaceScore, userReview);
 
     //TODO: Save data to database
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
```

Comment:
Logic looks good, I'll go back and merge this into the FormHandlerServlet on my end.